client/landing/gutenboarding/onboarding-block/signup-form/index.tsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
```

Comment:
I don't think we've used any lodash in Gutenboarding so far. Since it sounds like the ultimate goal is to move Gutenboarding into `/apps` (where we might not end up using lodash), would it make sense to replace `noop` with `() => undefined`?

Reply:
I thought this too, but we do use a few (`times`, `map` and `partition`). 

Agree that `noop` isn't really required.

Reply:
Ah, I missed that in my search! Thanks.

Reply:
I don't wanna start pulling in `lodash` if we don't have to! I feel like modern JS has enough goodies in it without the utility library. I really only pulled it in so I could shut up an eslint warning. Maybe `() => undefined` will do the trick though ...

Reply:
It does! Thanks for the tip @andrewserong 👍

Reply:
https://github.com/Automattic/wp-calypso/pull/37713#discussion_r348023578 😏